documentation/asciidoc/computers/compute-module/datasheet.adoc

```diff
@@ -13,6 +13,27 @@ There is also a KiCad PCB design set available:
 
 * https://datasheets.raspberrypi.com/cm4io/CM4IO-KiCAD.zip[Compute Module 4 IO Board KiCad files]
 
+[.whitepaper, title="Transitioning from CM3 to CM4", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-003469-WP/Transitioning-from-CM3-to-CM4.pdf]
+****
+This whitepaper is for those who wish to move from using a Raspberry Pi Compute Module (CM) 1 or 3 to a Raspberry Pi CM 4. There are several reasons why this might be desirable:
+• Greater computing power
+• More memory
+• Higher-resolution output up to 4Kp60
+• Better availability
+• Longer product life (last time buy not before January 2028)
+
+From a software perspective, the move from Raspberry Pi CM 1/3 to Raspberry Pi CM 4S is relatively painless, as a Raspberry Pi operating system (OS) image should work on all platforms. If, however, you are using a custom kernel, some things will need to be considered in the move. The hardware changes are considerable, and the differences are described in a later section.
```

Comment:
Still mentions CM 4S. And again, for consistency you should probably just use "Raspberry Pi OS". And "a later section" feels weird written here, when this is referring to a later section of the whitepaper rather than a later section of the documentation?